Mathlib/FieldTheory/Galois/Basic.lean

```diff
@@ -278,6 +278,81 @@ def galoisCoinsertionIntermediateFieldSubgroup [FiniteDimensional F E] [IsGalois
 
 end IsGalois
 
+section
+
+/-In this section we prove that the normal subgroups correspond to the Galois subextensions
+in the Galois correspondence and its related results.-/
+
+variable {K L : Type*} [Field K] [Field L] [Algebra K L]
+
+open IntermediateField
+
+open scoped Pointwise
+
+lemma AlgEquiv.restrictNormalHomKer (E : IntermediateField K L) [Normal K E]:
+    (restrictNormalHom E).ker = E.fixingSubgroup := by
+  ext σ
+  apply ((((mem_fixingSubgroup_iff (L ≃ₐ[K] L)).trans ⟨fun h ⟨x, hx⟩ ↦ Subtype.val_inj.mp <|
+    (restrictNormal_commutes σ E ⟨x, hx⟩).trans (h x hx) , _⟩).trans
+      AlgEquiv.ext_iff.symm)).symm
+  intro h x hx
+  have hs : ((restrictNormalHom E) σ) ⟨x, hx⟩ = σ • x :=
+    restrictNormal_commutes σ E ⟨x, hx⟩
+  exact hs ▸ (Subtype.val_inj.mpr (h ⟨x, hx⟩))
+
+namespace IsGalois
+
+variable (E : IntermediateField K L)
+
+/-- If `H` is a normal Subgroup of `Gal(L / K)`, then `fixedField H` is Galois over `K`. -/
+instance of_fixedField_normal_subgroup [IsGalois K L]
+    (H : Subgroup (L ≃ₐ[K] L)) [hn : Subgroup.Normal H] : IsGalois K (fixedField H) where
+  to_isSeparable := Algebra.isSeparable_tower_bot_of_isSeparable K (fixedField H) L
+  to_normal := by
+    apply normal_iff_forall_map_le'.mpr
+    rintro σ x ⟨a, ha, rfl⟩ τ
+    exact (symm_apply_eq σ).mp (ha ⟨σ⁻¹ * τ * σ, Subgroup.Normal.conj_mem' hn τ.1 τ.2 σ⟩)
+
+/-- If `H` is a normal Subgroup of `Gal(L / K)`, then `Gal(fixedField H / K)` is isomorphic to
+`Gal(L / K) ⧸ H`. -/
+noncomputable def normalAutEquivQuotient [FiniteDimensional K L] [IsGalois K L]
+    (H : Subgroup (L ≃ₐ[K] L)) [Subgroup.Normal H] :
+    (L ≃ₐ[K] L) ⧸ H ≃* ((fixedField H) ≃ₐ[K] (fixedField H)) :=
+  (QuotientGroup.quotientMulEquivOfEq ((fixingSubgroup_fixedField H).symm.trans
+  (AlgEquiv.restrictNormalHomKer (fixedField H)).symm)).trans <|
+  QuotientGroup.quotientKerEquivOfSurjective (restrictNormalHom (fixedField H)) <|
+  restrictNormalHom_surjective L
+
+lemma normalAutEquivQuotient_apply [FiniteDimensional K L] [IsGalois K L]
+    (H : Subgroup (L ≃ₐ[K] L)) [Subgroup.Normal H] (σ : (L ≃ₐ[K] L)) :
+    normalAutEquivQuotient H (QuotientGroup.mk σ) = (restrictNormalHom (fixedField H)) σ := rfl
+
+open scoped Pointwise
+
+theorem fixingSubgroup_conjugate_of_map (σ : L ≃ₐ[K] L) :
+    (E.map σ).fixingSubgroup = (MulAut.conj σ) • E.fixingSubgroup := by
+  ext τ
+  rw [IntermediateField.fixingSubgroup, mem_fixingSubgroup_iff]
```

Comment:
Maybe there should be a `simp`-lemma `IntermediateField.mem_fixingSubgroup_iff` to avoid having to unfold the definition of `IntermediateField.fixingSubgroup`.